packages/client/api/src/util.ts

```diff
@@ -0,0 +1,32 @@
+import type { IFluenceClient } from '@fluencelabs/interfaces';
+
+const getPeerFromGlobalThis = (): IFluenceClient | undefined => {
+    // @ts-ignore
+    return globalThis.defaultPeer;
+};
+
+// TODO: DXJ-271
+const REJECT_MESSAGE = 'You probably forgot to add script tag. Read about it here: ';
+
+/**
+ * Wait until the js client script it loaded and return the default peer from globalThis
+ */
+export const getDefaultPeer = (): Promise<IFluenceClient> => {
+    return new Promise((resolve, reject) => {
+        let interval: NodeJS.Timer | undefined;
+        let hits = 50;
```

Comment:
seems arbitrary. What it depends on? Do you think each lib user will have roughly the same requirements ceiling of 5 seconds loading time?

Reply:
I've added a couple of comments